packages/fxa-admin-server/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "FxA GraphQL Admin Server",
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "yarn prebuild && yarn nest build && cp ./src/config/*.json ./dist/config",
+    "build": "yarn prebuild && yarn nest build && yarn copy-src && yarn copy-config",
+    "copy-src": "mv ./dist/src/* ./dist/.",
```

Comment:
By adding "scripts" to as a source folder it changes the state of dist folder. Essentially there is now a dist/src and dist/scripts. To keep things the way they were, we will move the contents of dist/src to dist.